osu.Game/Screens/SelectV2/PanelGroup.cs

```diff
@@ -45,6 +45,7 @@ private void load()
             {
                 AlwaysPresent = true,
                 RelativeSizeAxes = Axes.Y,
+                Alpha = 0f,
```

Comment:
Noticed in group panel tests that the chevron initially goes from visible to hidden state, which is wrong since the group defaults to contracted state.

Before:

https://github.com/user-attachments/assets/00dcc618-9db6-4c7f-bc55-a7066c5421e7

After:

https://github.com/user-attachments/assets/7135ca54-f50c-4903-9b56-d360282f9f5e